src/ai/backend/manager/models/keypair.py

```diff
@@ -60,6 +64,15 @@
     "verify_dotfile_name",
 )
 
+_rolling_count_script = """
+local access_key = KEYS[1]
+local now = tonumber(ARGV[1])
+local window = tonumber(ARGV[2])
+if redis.call('EXISTS', access_key) == 1 then
+    redis.call('ZREMRANGEBYSCORE', access_key, 0, now - window)
+end
+return redis.call('ZCARD', access_key)
+"""
```

Comment:
I think it should be a just simple call of `ZCARD` instead of running a full script, as the API request to get this value is already taken into account for rate limiting.